src/amo/components/AddonHead/index.js

```diff
@@ -205,26 +175,29 @@ export class AddonHeadBase extends React.Component<InternalProps> {
     }
 
     return (
-      <Helmet titleTemplate={null}>
-        <title>{this.getPageTitle()}</title>
-
-        <link
-          rel="canonical"
-          href={getCanonicalURL({ locationPathname, _config })}
-        />
-        {this.renderAlternateLinks()}
-
-        <meta name="description" content={this.getPageDescription()} />
-        <meta name="date" content={addon.created} />
-        {addon.last_updated && (
-          <meta name="last-modified" content={addon.last_updated} />
-        )}
-        {this.renderMetaOpenGraph()}
-
-        <script type="application/ld+json">
-          {JSON.stringify(getAddonJsonLinkedData({ addon }))}
-        </script>
-      </Helmet>
+      <React.Fragment>
+        <Helmet titleTemplate={null}>
+          <title>{this.getPageTitle()}</title>
+
+          <link
+            rel="canonical"
```

Comment:
Would it make sense to move this into the new component you created? It seems like when rendering alternate links that you'd always want to include the canonical link, too. It also seems like you could safely build this within `HrefLang` with the `to` property.

Reply:
`HeadLinks` wfm